.circleci/config.yml

```diff
@@ -92,6 +92,7 @@ workflows:
             - install
       - test:
           requires:
+            - build
```

Comment:
Do we need this?

Reply:
Probably not. That's left over when I had the first commits to the repo